### DISCUSSION TITLE
How "score" fits into the ecosystem with popular IaC tools like Terraform & Pulumi?

### DISCUSSION THREAD

**Discussion by amjathsharief** [score-spec/spec#55] [Q&A]
How is score related to popular IaC tools like terraform & pulumi? 

Do you think we will have something like "score-terraform" which will generate terraform code to provision infrastructure required to run a micro service?

Also "score-pulumi" which will generate pulumi yaml file to provision infrastructure required to run a micro service?

Or is my understanding completely wrong w.r.t the goals of score? Please clarify.

**Comment by sujaya-sys** [comment]
Hey @amjathsharief, sorry for the late response, I somehow totally missed your question!

> Do you think we will have something like "score-terraform" which will generate terraform code to provision infrastructure required to run a micro service?

Score is for defining workloads rather than infrastructure, so there is not the ability to convert yaml into HCL for example.

> How is score related to popular IaC tools like terraform & pulumi?

Developers can define resource dependencies in their Score spec, which might be resolved by IaC tools down the line but this is nothing Score is directly involved with as described here: https://docs.score.dev/docs/concepts/dependencies/
